_includes/v21.2/sql/replication-zone-patterns-to-multiregion-sql-mapping.md

```diff
@@ -1,5 +1,5 @@
-| Replication Zone Pattern                         | Multi-Region SQL                                                                                                                            |
+| Replication Zone Pattern                         | Multi-Region SQL                                                                                                                                    |
 |--------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [Duplicate indexes](topology-duplicate-indexes.html)                  | [`GLOBAL` tables](global-tables.html)                                                                                                                       |
-| [Geo-partitioned replicas](topology-geo-partitioned-replicas.html)        | [`REGIONAL BY ROW` tables](regional-tables.html#regional-by-row-tables) with [`ZONE` survival goals](multiregion-overview.html#surviving-zone-failures)     |
-| [Geo-partitioned leaseholders](topology-geo-partitioned-leaseholders.html) | [`REGIONAL BY ROW` tables](regional-tables.html#regional-by-row-tables) with [`REGION` survival goals](multiregion-overview.html#surviving-region-failures) |
+| [Duplicate indexes](../v20.2/topology-duplicate-indexes.html)                  | [`GLOBAL` tables](global-tables.html)                                                                                                                       |
+| [Geo-partitioned replicas](../v20.2/topology-geo-partitioned-replicas.html)        | [`REGIONAL BY ROW` tables](regional-tables.html#regional-by-row-tables) with [`ZONE` survival goals](multiregion-overview.html#surviving-zone-failures)     |
```

Comment:
Guessing this is for the same reasons as above, and explanation will be the same

Reply:
Yep, same explanation.